### PR TITLE
Update kubeflow banner to 1.7

### PR DESCRIPTION
**Pull request by carkod**
## Done

Add Kubeflow beta banner, [as per request](https://docs.google.com/document/d/19_9cUiyjxWXPvLaZc0_DOglsCQwlJFW8gET7sZt7IYI/edit)

## QA

- Check banner shows on
https://charmed-kubeflow-io-159.demos.haus/
https://charmed-kubeflow-io-159.demos.haus/tutorials

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2205?atlOrigin=eyJpIjoiNmE0MzhkYmY1MmIxNDg3Y2FiZmM0MGYzMWEyYTkxMDgiLCJwIjoiaiJ9

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/partial/_beta-banner.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
   <div class="p-notification--information is-inline">
     <div class="p-notification__content">
       <h5 class="p-notification__title">
-        The Charmed Kubeflow 1.6 beta is here:
+        The Charmed Kubeflow 1.7 beta is here:
       </h5>
       <p class="p-notification__message">
         <code>juju deploy kubeflow --channel 1.6/beta</code><br /> Read more in our
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/partial/_beta-banner.html:8]: Yes, the change is from 1.6 to 1.7. You reviewed the old file :)

File: templates/partial/_beta-banner.html [hunk 2]
```diff
@@ -2,12 +2,12 @@
   <div class="p-notification--information is-inline">
     <div class="p-notification__content">
       <h5 class="p-notification__title">
-        The Charmed Kubeflow 1.6 beta is here:
+        The Charmed Kubeflow 1.7 beta is here:
```
Comment (carkod) [inline templates/partial/_beta-banner.html:5]: @pedoch this is the new file change

File: templates/partial/_beta-banner.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (carkod) [reply templates/partial/_beta-banner.html:8]: Oh yeah, I see what you mean

**Review by pedoch — APPROVED** [review]
LGMT! 👍

**Pull request merged**